btx/diagnostics/geom_opt.py

```diff
@@ -167,19 +162,17 @@ def finalize(self):
                                                self.distance,
                                                self.diagnostics.psi.get_pixel_size())[0] # in Angstrom
             
-    def deploy_geometry(self, outdir):
+    def deploy_geometry(self, outdir, pv_camera_length):
```

Comment:
should we set it to None by default?

Reply:
Thanks for catching that! It was handled okay in tasks.py but could have posed problems if using the class outside of that. Fixed in the latest commit.